src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,2562 @@
+        "iataCode": {
+          "type": "string",
+          "description": "The IATA code for the location"
+        },
+        "id": {
+          "type": "string",
+          "description": "The id of the location"
+        },
+        "latitude": {
+          "type": "number",
+          "description": "The latitude of the location"
+        },
+        "lnKey": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The key of the location name"
+        },
+        "locationCode": {
+          "type": "string",
+          "description": "The code of the location"
+        },
+        "locationType": {
+          "type": "string",
+          "description": "The type of the location"
+        },
+        "longitude": {
+          "type": "number",
+          "description": "The longitude of the location"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the location"
+        }
+      }
+    },
+    "PageResultOfResourceOfRequest": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Request"
+          }
+        },
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Link"
+          }
+        }
+      }
+    },
+    "Request": {
+      "type": "object",
+      "properties": {
+        "approvalLimitDate": {
+          "type": "string",
+          "description": "The date by which the Request must be approved. This element appears only when integrated with Concur Travel"
+        },
+        "approvalStatus": {
+          "description": "The approval status of the Request",
+          "$ref": "#/definitions/ApprovalStatus"
+        },
+        "approved": {
+          "type": "boolean",
+          "description": "Indicates whether this Request is approved"
+        },
+        "authorizedDate": {
+          "type": "string",
+          "description": "The date when the Request was authorized"
+        },
+        "businessPurpose": {
+          "type": "string",
+          "description": "The business purpose of the Request"
+        },
+        "canceledPostApproval": {
+          "type": "boolean",
+          "description": "Indicates whether this Request was canceled post approval"
+        },
+        "closed": {
+          "type": "boolean",
+          "example": false,
+          "description": "Indicates whether the Request is closed"
+        },
+        "comment": {
+          "type": "string",
+          "description": "The last comment attached to this Request"
+        },
+        "comments": {
+          "description": "Historical comments header to the Request",
+          "$ref": "#/definitions/ResourceLink"
+        },
+        "creationDate": {
+          "type": "string",
+          "description": "For approved Request, the date at which the approval process was completed"
```

Comment:
@quancheng-zhao 
```suggestion
          "description": "Date when the request was approved"
```

Reply:
Date when the request was created (in the format yyyy-MM-dd'T'HH:mm:ss.SSS'Z')